packages/graphql/src/translate/cypher-builder/Pattern.ts

```diff
@@ -157,4 +157,15 @@ export class Pattern<T extends MatchableElement = any> implements CypherCompilab
 
         return padLeft(stringifyObject(paramValues));
     }
+
+    private getNodeLabelsString(node: NodeRef): string {
+        const escapedLabels = node.labels.map(escapeLabel);
+        if (escapedLabels.length === 0) return "";
+        return `:${escapedLabels.join(":")}`;
+    }
+
+    private getRelationshipLabelsString(relationship: RelationshipRef): string {
```

Comment:
question (non-blocking): Do you also think `getRelationshipTypesString` is more accurate in this case?
```suggestion
    private getRelationshipTypesString(relationship: RelationshipRef): string {
```

Reply:
100% agree